### ISSUE TITLE
php: Call to a member function getUID() on a non-object / trashbin.php

### ISSUE THREAD

**Issue by taradiddles** [nextcloud/server#195]
### Steps to reproduce

runuser -u apache php /var/www/nextcloud/cron.php

throws the following error:

PHP Fatal error:  Call to a member function getUID() on a non-object in /var/www/nextcloud/apps/admin_audit/lib/actions/trashbin.php on line 49

### Server configuration
**Operating system**: centos 7.2.1511 (core), updated.

**Web server:** httpd-2.4.6-40.el7.centos.1.x86_64

**Database:** mariadb-5.5.47-1.el7_2.x86_64

**PHP version:**  php-5.4.16-36.1.el7_2.1.x86_64

**Nextcloud version:** 9.0.50, updated from owncloud (as per blog post)

**Where did you install Nextcloud from:** website

**Signing status (Nextcloud 9.0 and above):** ?

**config:**

```
{
    "system": {
        "updatechecker": false,
        "instanceid": "ocuyvyclnt4t",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "ssl.example.com"
        ],
        "datadirectory": "\/srv\/oc_data",
        "overwrite.cli.url": "https:\/\/ssl.example.com\/owncloud",
        "dbtype": "mysql",
        "version": "9.0.50.0",
        "logtimezone": "UTC",
        "installed": true,
        "mail_from_address": "noreply",
        "mail_smtpmode": "smtp",
        "mail_domain": "example.com",
        "mail_smtphost": "smtp.example.com",
        "mail_smtpport": "25",
        "mail_smtpauthtype": "PLAIN",
        "asset-pipeline.enabled": true,
        "enable_avatars": false,
        "session_lifetime": 14400,
        "remember_login_cookie_lifetime": 172800,
        "trashbin_retention_obligation": "auto,2",
        "maintenance": false,
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "singleuser": false,
        "htaccess.RewriteBase": "\/owncloud"
    }
}
```

**apps**: 
```
Enabled:
  - activity: 2.2.1
  - admin_audit: 1.0.0
  - comments: 0.2
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - files: 1.4.4
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - gallery: 14.5.0
  - notifications: 0.2.3
  - systemtags: 0.2
Disabled:
  - encryption
  - external
  - federation
  - files_external
  - firstrunwizard
  - provisioning_api
  - templateeditor
  - updatenotification
  - user_external
  - user_ldap
```

**Comment by MorrisJobke** [comment]
cc @schiessle

**Comment by ttcc-gh** [comment]
Same issie here:

Debian 8
Nextcloud 9.0.51 (owncloud update)

**Comment by LukasReschke** [comment]
@schiessle Please backport https://github.com/nextcloud/server/pull/164 as advised.

**Comment by taradiddles** [comment]
That was fast...
I did search for existing issues before posting that one, but somehow kept the issue:open tag in the search field. I now see it's a dup of (closed) issue #143. Sorry about that.
Keep up the good work :)
